modules/context/form.go

```diff
@@ -65,3 +65,8 @@ func (ctx *Context) FormOptionalBool(key string) util.OptionalBool {
 	v = v || strings.EqualFold(s, "on")
 	return util.OptionalBoolOf(v)
 }
+
+func (ctx *Context) SetFormValue(key, value string) {
```

Comment:
To match `FormString`, it should be `SetFormString`

And `Form.Set` is also `Set(key, value string)`, string

Reply:
done.